fix(cli): alias prefix can be longer than 1 character

jethack23 opened the pull request:
## Situation
I'm using `/shadcn` as alias prefix with below configurations.
### components.json includes
```
"aliases": {
    "components": "/shadcn/components",
    "utils": "/shadcn/lib/utils",
    "ui": "/shadcn/components/ui",
    "lib": "/shadcn/lib",
    "hooks": "/shadcn/hooks"
  }
```
### tsconfig.json includes
```
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "/shadcn/*": ["./shadcn/*"]
    },
```
It worked fine for me with former version.

## Bug with new cli version
When I added new component with `npx shadcn@latest add radio-group`, what I was expected was import line of
```
import { cn } from "/shadcn/lib/utils"
```
But what I got was import line of
```
import { cn } from "//lib/utils"
```
I guess this is the same issue that #4691 describes

## What I did
- [x] Added new test case with alias `/shadcn`
- [x] Run test with `pnpm test` as contributing guidelines says and checked it fails
- [x] Edit the getsTsConfigAliasPrefix function in "packages/cli" directory
- [x] Run test again with `pnpm test` and checked it passed

shadcn reviewed (changes requested):
Agreed. Can we add a test for it please?

jethack23 commented:
I thought I already added the test for it in the commit 1cc35ac in pull request.
- project folder for the testcase `packages/cli/test/fixtures/general-alis` (which contains tsconfig.json only)
- and I added the testcase in `packages/cli/test/utils/get-ts-config-alias-prefix.test.ts`
- and check it fails in main branch
- and check it passes in my branch

If I have misunderstood something, can you explain more details about "adding a test for it" please?
It will help for me to make appropriate changes.

jethack23 commented:
@shadcn As I read your comment on #4733, I figured out that new cli package directory is `packages/shadcn`, not `packages/cli`.

So I made these changes
- New test project directory for it in `packages/shadcn/test/fixtures/frameworks/general-alias` by copying and paste `packages/shadcn/test/fixtures/frameworks/next-app` and change its tsconfig.json
- New test code in `packages/shadcn/test/utils/get-project-info.test.ts`
- Same fix in `packages/shadcn/src/utils/get-project-info.ts`
- Change `CONTRIBUTING.md` so that it specify `packages/shadcn` as "any changes to the CLI should be made" and updated Structure section.

ravenkim commented:
same issue 
#4737

jethack23 commented:
@seanyasno I'm using `npx shadcn-ui@0.8.0 add SOMETHING` for temporary solution.
@shadcn Can you give any feedback please?
